### ISSUE TITLE
Farmers fall through Farm blocks

### ISSUE THREAD

**Issue by Konyv** [TangoTek/TekTopia-Community#262]
**Describe the bug**
Farms sometimes fall through farm blocks. 
The farm blocks have an air block beneath them. 
I assume this is because farm blocks are smaller than full blocks. 

**To Reproduce**
1. Have farmers and farms
2. wait
3. Eventually one glitches down.

**Expected behavior**
We expect the farm to not get stuck in a block.

**Screenshots**
![2019-06-01_19 22 52](https://user-images.githubusercontent.com/51104278/58756899-68e8f780-84c6-11e9-810d-a1af7701b2a4.png)
![2019-06-01_19 23 00](https://user-images.githubusercontent.com/51104278/58756900-69818e00-84c6-11e9-80ad-21c3a9013a13.png)
![2019-06-01_19 23 32](https://user-images.githubusercontent.com/51104278/58756901-6b4b5180-84c6-11e9-8680-ba38a6614aaf.png)
![2019-06-01_19 23 41](https://user-images.githubusercontent.com/51104278/58756902-6c7c7e80-84c6-11e9-875b-9ab3a00efae9.png)

** Server/Location **
what if server 1

**Additional context**
I fixed it by digging out under them and then removing all air blocks beneath the farm blocks.

**Comment by JohnyDL** [comment]
I'm not sure if this is standing on farmland when it's hoed or standing on crops when they're harvested and unhoed and somehow they end up in that dirt block
I think my farmers are suddenly working harder and lots of falling through the floor farmer got damaged today

**Comment by Konyv** [comment]
I think this would be standing on crops when they're harvested and unhoed and somehow they end up in that dirt block.

That farm has been there for awhile.

**Comment by PunchAndDunk** [comment]
Yes, this has happened to me as well. Any one block farmland they eventually fall through.

**Comment by TheGreatNaj** [comment]
Not sure if this is a mod bug tbh, because similar happens in vanilla. Like when a player tramples a crop turning it back to dirt and they glitch out because they're technically inside the block. Its more about the block than the villager.

**Comment by ghost** [comment]
![2019-06-25_00 22 13](https://user-images.githubusercontent.com/51976349/60813233-46858080-a1ce-11e9-97d4-58ebf0930068.png)
There was two block deep gap under my farmland blocks

**Comment by MorsDux** [comment]
My first occurence was after I got a second farmer, couldnt catch it but I think it happens when a farmer botches a harvest where another farmer (or self) stands. This makes the block higher than the farm block and so the villager falls through.
